### ISSUE TITLE
Reproduction of dev2.0 is 2-3 percent lower than official results

### ISSUE THREAD

**Issue by evakasch** [HuangJunJie2017/BEVDet#135]
I set up the environment and run this code using 8 GPUS in config [BEVDet-R50-4D-Depth-CBGS](https://github.com/HuangJunJie2017/BEVDet/blob/dev2.0/configs/bevdet/bevdet4d-r50-depth-cbgs.py).
The reproduced NDS/mAP on v1.0-val dataset is: 49.89 / 37.44, which is much lower than your results 52.3 / 40.2.

The last loss value 10.7642 is close to your log file 10.7559. My mmdet3d version is using this code repository--1.0.0 rc4

I use epoch_20.pth to evaluate the val result, the detail results are
mAP: 0.3744
mASE: 0.5933
mASE: 0.2815
mAOE: 0.4938
mAVE: 0.3165
mAAE: 0.1981
NDS: 0.4989

The gap is really big and can hardly be treated as random noise. 

Is the official log/results corresponding to dev-2.0 codes/configs?

**Comment by HuangJunJie2017** [comment]
@evakasch use epoch_20_ema.pth instead?

**Comment by evakasch** [comment]
Thanks for your help! The result using epoch_20_ema.pth is much higher.
Detail results are 
mAP: 0.3974
mATE: 0.5739
mASE: 0.2772
mAOE: 0.4616
mAVE: 0.2915
mAAE: 0.1970
NDS: 0.5186

The reproduced results are 0.5 percent lower than your official results. Could this be treated as random noise? Or is there still other operations missing / done incorrectly?

**Comment by HuangJunJie2017** [comment]
@evakasch  it can be treated as random noise if you haven't modified the release code.

**Comment by HuangJunJie2017** [comment]
@evakasch 
I found that the released code will start training with temporal later than the released model/log
as we can see the loss of velocity drop rapidly at epoch 4 in the released log instead of epoch 5 in retrained log……
maybe this will lead to performance degeneration 52.2->51.9
I will revise this after searching for the optimal hyperparameter.
